middleman/media_responses.py

```diff
@@ -0,0 +1,176 @@
+                    else:
+                        management_room_text = f"{media_name[self.media_type]} delivered back to the sender in room {message['room_id']}."
+                    logger.info(f"{media_name[self.media_type]} {self.event.event_id} relayed back to the original sender")
+                else:
+                    if self.config.confirm_reaction:
+                        management_room_text = self.config.confirm_reaction_fail
+                    else:
+                        management_room_text = f"Failed to send {media_name[self.media_type]} back to sender:" \
+                            f"{response.message if isinstance(response, RoomSendError) else response}"
+                    logger.warning(management_room_text)
+                # Confirm in management room
+                if self.config.confirm_reaction:
+                    await send_reaction(
+                        self.client,
+                        self.room.room_id,
+                        self.event.event_id,
+                        management_room_text
+                    )
+                else:
+                    await send_text_to_room(
+                        self.client,
+                        self.room.room_id,
+                        management_room_text,
+                        True,
+                    )
+            else:
+                logger.debug(
+                    f"Skipping {media_name[self.media_type]} {self.event.event_id} which is not a reply to one of our relay messages",
+                )
+        else:
+            logger.debug(f"Skipping {self.event.event_id} reply {media_name[self.media_type]}")
+
+    def is_mention_only_room(self, identifiers: List[str], is_named: bool) -> bool:
+        """
+        Check if this room is only if mentioned.
+        """
+        if self.config.mention_only_always_for_named and is_named:
+            return True
+        for identifier in identifiers:
+            if identifier in self.config.mention_only_rooms:
+                return True
+        return False
+
+    async def process(self):
+        """
+        Process media.
+        - if management room, identify replies and forward back to original messages.
+        - anything else, relay to management room.
+        """
+        if self.room.room_id == self.config.management_room_id:
+            await self.handle_management_room_media()
+        else:
+            await self.relay_to_management_room()
+
+    async def relay_to_management_room(self):
+        """Relay to the management room."""
+        # First check if we want to relay this
+        if self.is_mention_only_room([self.room.canonical_alias, self.room.room_id], self.room.is_named):
+            # skip media in mention only rooms for now
+            logger.debug(f"Skipping {media_name[self.media_type]} %s in room %s as it's set to only relay on mention and we were "
+                         "not mentioned.", self.event.event_id, self.room.room_id)
```

Comment:
```suggestion
            logger.debug(f"Skipping {media_name[self.media_type]} %s in room %s as it's set to only relay on mention and mentioning in "
                         "a media reply is not supported.", self.event.event_id, self.room.room_id)
```

Technically this debug message is not true since it's not possible to even mention someone in a media event :)